### DISCUSSION TITLE
Express Form for Craft 2

### DISCUSSION THREAD

**Discussion by rose02222** [solspace/craft-express-forms#68] [Questions & Answers] [answered]
Hi! 

Do you have an Express Form plugin that is compatible with Craft 2?
I'm trying to use it but it says the compatibility is Craft 3.

Thanks in advance!

### Additional info

- Express Forms edition & version: N/A
- Craft version: PRO Craft CMS 2.7.4 
- PHP version:5.6.40
--

- Database driver & version:
- Other plugins & versions (if applicable):

**Comment by kjmartens** [comment, marked as answer]
Thanks for your interest in Express Forms, but it is Craft 3+ only. 🙂

**Comment by rose02222** [comment]
Thank you for your answer. Sorry I'm new with craft and wondering because I'm seeing most of the plugins compatibility was on Craft 3. Does that mean they are not compatible with Craf2?

**Comment by kjmartens** [comment]
@rose02222 Craft 2 is very old now. Many of the popular plugins you see (such as Freeform) do have a Craft 2 version, but that's only because they existed on Craft 2 before v3 came out. But I would also imagine most plugin developers are no longer providing support for their Craft 2 versions at this point. Is there any reason you cannot upgrade to Craft 3? :)

**Comment by rose02222** [comment]
@kjmartens  thanks for sharing that. We are still figuring it out how to upgrade on Craft 3, it seems it would take a lot of work :D
